### PR TITLE
Add sample playbooks

### PR DESCRIPTION
**Pull request by DenDen039**
##### SUMMARY
Adds samples of playbooks

##### ISSUE TYPE
- New Playbooks Pull Request

##### ADDITIONAL INFORMATION
Adds playbooks from https://github.com/PureStorage-OpenConnect/fusion-client-devkit

### CODE REVIEW

**Comment by DenDen039** [comment]
@sdodsley can you review a structure of playbooks, please? I tried to find examples in other collections, but our case is a bit different. I would say it is something between roles and playbooks.

**Review by sdodsley — DISMISSED** [review]
Nothing leaps out at me as looking incorrect.
I assume you have tested all of these....

**Comment by DenDen039** [comment]
> Nothing leaps out at me as looking incorrect. I assume you have tested all of these....

Forgot to say that I have deleted roles from playbooks directory. Because of Ansible Docs https://docs.ansible.com/ansible/latest/dev_guide/developing_collections_structure.html#playbooks-directory

File: playbooks/files/initiator_random_write.fio [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[write-verify]
```
Comment (pure-blim) [inline playbooks/files/initiator_random_write.fio:1]: Can we move the whole `files/` directory into `sample_production`. Only the playbooks in sample_production make use of this file

File: playbooks/inventory/group_vars/all [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# read from OS vars
```
Comment (pure-blim) [inline playbooks/inventory/group_vars/all:1]: Similarly, can we move the whole `inventory` into `sample_production`?

**Review by pure-blim — COMMENTED** [review]
Looks mostly right. What I had in mind was something like [this](https://github.com/ansible/ansible-examples), where the top-level (in this case playbooks/), contain a number of different example scenarios. We have 2 scenarios -- simple and sample_production.

And once you open up sample_production, the directory layout should match how it might be deployed, so you should see subdir like inventory, files, group_vars, etc, according to [best practices](https://docs.ansible.com/ansible/2.8/user_guide/playbooks_best_practices.html#directory-layout).

Thus, my only comment is can we move files/ and inventory/ from the top-level to subdirs of sample_production?

**Review by pure-blim — APPROVED** [review]

**Review by sdodsley — APPROVED** [review]

**Review by andreepyro — APPROVED** [review]

**Pull request merged**